force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -57,11 +60,16 @@ public with sharing class FlowControlLoopsCollections {
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
         Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+        if (num1 == num2) {
+            largestNum = num1;
+        } else if (num1 > num2) {
+            largestNum = num1;
+        } else if (num1 < num2) {
+            largestNum = num2;
+        }
+        return largestNum; 
```

Comment:
You can simplify this, but I think actually the way it is now is very readable. Some might combine the first if statement with one of the later ones using a >= operator, but it's preference.